Doppler.AccountPlans.Test/CalculateUpgradeCostTest.cs

```diff
@@ -70,6 +70,37 @@ public void CalculateUpgradeCostHelper_Monthly_method_should_return_correct_valu
             Assert.Equal(0, result.DiscountPaymentAlreadyPaid);
         }
 
+        [Fact]
+        public void CalculateUpgradeCostHelper_Monthly_method_should_fail_when_monthly_plan_is_not_detected()
```

Comment:
Should fail? I am missing something?

```suggestion
        public void CalculateUpgradeCostHelper_Monthly_method_should_not_fail_when_monthly_plan_is_not_detected()
```

Reply:
Sorry, I should not have uploaded that test